config/routes.rb

```diff
@@ -34,6 +34,7 @@
   get 'samples/:id/assembly/:taxid', to: 'samples#assembly'
   get 'samples/:id/alignment/:taxon_info', to: 'samples#show_taxid_alignment'
   get 'samples/:id/alignment_viz/:taxon_info', to: 'samples#show_taxid_alignment_viz'
+  get 'show_sample_names', to: 'samples#show_sample_names'
```

Comment:
maybe for hierarchy 'samples/show_names'

Reply:
ok